frontend/packages/console-dynamic-plugin-sdk/src/shared-modules.ts

```diff
@@ -21,6 +23,12 @@ export const sharedVendorModules = [
  */
 export const overrideSharedModules = (entryModule: RemoteEntryModule) => {
   entryModule.override({
+    '@openshift-console/dynamic-plugin-sdk': async () => () =>
+      // eslint-disable-next-line
```

Comment:
I think we don't need those `eslint-disable-next-line` comments here anymore.

Reply:
removed